<launch>
  <!-- master DAVIS -->
  <node ns="davis_left" name="davis_ros_driver" pkg="davis_ros_driver" type="davis_ros_driver" output="screen">
    <param name="serial_number" type="str" value="84010032" />
    <rosparam command="load" file="$(find davis_ros_driver)/config/standard.yaml" />
    <param name="master" value="True"/>

    <!-- timeout before resetting time -->
    <param name="reset_timestamps_delay" value="2.0"/>
  </node>

  <!-- slave DAVIS -->
  <node ns="davis_right" name="davis_ros_driver" pkg="davis_ros_driver" type="davis_ros_driver" output="screen">
    <param name="serial_number" type="str" value="84010059" />
    <rosparam command="load" file="$(find davis_ros_driver)/config/standard.yaml" />
    <param name="master" value="False"/>

    <!-- get reset signal from master -->
    <remap from="reset_timestamps" to="/davis_left/reset_timestamps"/>
  </node>
</launch>
